test/integration/consul-container/go.mod

```diff
@@ -0,0 +1,30 @@
+module github.com/hashicorp/consul/integration/consul-container
+
+go 1.16
```

Comment:
could we go w 1.18 for this since it's a new module?

Reply:
This is not the version used, this the lowest version that can be compatible with the module. We actually use the same version as the one we use to compile Consul to run this, which is for now 1.17